bugsnag-android-core/src/main/java/com/bugsnag/android/BugsnagException.java

```diff
@@ -39,11 +48,23 @@ public BugsnagException(@NonNull String name,
             this.message = ((BugsnagException) exc).getMessage();
             this.name = ((BugsnagException) exc).getName();
             this.type = ((BugsnagException) exc).getType();
+        } else if (exc instanceof JsonStream.Streamable) {
```

Comment:
The first conditional case might by lossy. If `exc instanceof BugsnagException` and it wraps a streamable, the streamable content might be lost. However, every `BugsnagException`	is an instance of `Streamable` which could be used directly.